examples/evaluator-rules/src/main/resources/example.rules.kts

```diff
@@ -141,9 +141,9 @@ fun RuleSet.copyleftInSourceRule() = packageRule("COPYLEFT_IN_SOURCE") {
 
         val message = if (licenseSource == LicenseSource.DETECTED) {
             "The ScanCode copyleft categorized license $license was ${licenseSource.name.lowercase()} " +
-                    "in package ${pkg.id.toCoordinates()}."
+                    "in package ${pkg.pkg.id.toCoordinates()}."
         } else {
-            "The package ${pkg.id.toCoordinates()} has the ${licenseSource.name.lowercase()} ScanCode copyleft " +
+            "The package ${pkg.pkg.id.toCoordinates()} has the ${licenseSource.name.lowercase()} ScanCode copyleft " +
```

Comment:
If the rule class would still have separate properties (or just getters) for `pkg` and `curations` the rules.kts code IMO would be cleaner. Would you agree?

Reply:
No, I would actually not agree. `curations` is not an entity that's usually dealt with on its own in ORT's code base, they always come with an accompanying `Package` in the context of a `CuratedPackage`. As such I see no value in splitting something up here that belongs together, and that is always dealt with together in the rest of the code base.